README.md

```diff
@@ -8,6 +8,7 @@
 [![Ethereum Consensus Spec v1.1.10](https://img.shields.io/badge/ETH%20consensus--spec-1.1.10-blue)](https://github.com/ethereum/consensus-specs/releases/tag/v1.1.10)
 ![ES Version](https://img.shields.io/badge/ES-2021-yellow)
 ![Node Version](https://img.shields.io/badge/node-20.x-green)
+[![codecov](https://codecov.io/gh/ChainSafe/lodestar/graph/badge.svg?token=kHAUjtLsDG)](https://codecov.io/gh/ChainSafe/lodestar)
```

Comment:
Is the token required? Looks like it works without.

Reply:
I copied the markdown directly from their interface, but for public repo don't need the token.